app/src/main/java/com/dmonzonis/nookpendium/Record.kt

```diff
@@ -28,10 +28,6 @@ class Recordset(private val totalRecords: List<Record>) {
         return records
     }
 
-    fun applyFilterByMonth(month: Int): List<Record> {
```

Comment:
this is still possible with the new menu right?

Reply:
yes, it's just that the filter lambda is passed by the FilterManager, so Recordset just applies the filter to the records